package.json

```diff
@@ -36,9 +36,10 @@
   "version": "0.1.0",
   "devDependencies": {
     "are-you-es5": "^2.1.2",
+    "prettier": "3.3.3",
     "standard": "^17.0.0",
```

Comment:
Prettier is recommended in the upgrade of stylelint-config-gds to cover stylesheet files, but it can also cover various other file types - including js, where we are currently using standard. We could replace standard with prettier for js, although there are a few things that they disagree on - I've not done so here to reduce the size of the PR, but could raise another if people are keen.